hw4/03-form.js

```diff
@@ -1,18 +1,32 @@
-const express = require('express');
+const express = require("express");
 const app = express();
 const port = process.env.PORT || 5001;
 
 // Use middleware static() to serve all static files in the given folder
-app.use(express.static('public'));
+app.use(express.static("public"));
 
 // Use middleware urlencoded() to parse an incoming request with a urlencoded payload and return an objectß
 app.use(express.urlencoded({ extended: false }));
 
 // POST request
-app.post('', (req, res) => {
+app.post("/submit", (req, res) => {
```

Comment:
Form looks nice. I don't get any errors when loading and the output on the following page looks great! The only thing I  notice is that there is no validation, but I'm not sure that was required for this assignment anyway!
Looking good, imo.